dpd/p4/sidecar.p4

```diff
@@ -210,24 +320,25 @@ control Services(
 			drop_bad_ping;
 			forward_from_userspace;
 			forward_to_userspace;
-			multicast_inbound;
+			mcast_inbound_link_local;
 			NoAction;
 		}
 
 		const entries = {
-			( 0, false, false, true, _, _, false, true, true, false, ICMP_ECHOREPLY, 0 ) : forward_to_userspace;
-			( 0, false, false, true, _, _, false, true, true, false, ICMP_ECHOREPLY, _ ) : drop_bad_ping;
-			( 0, false, false, true, _, _, false, true, true, false, ICMP_ECHO, 0 ) : ping4_reply;
-			( 0, false, false, true, _, _, false, true, true, false, ICMP_ECHO, _ ) : drop_bad_ping;
-			( 0, false, false, true, _, _, false, true, false, true, ICMP6_ECHOREPLY, 0 ) : forward_to_userspace;
-			( 0, false, false, true, _, _, false, true, false, true, ICMP6_ECHOREPLY, _ ) : drop_bad_ping;
-			( 0, false, false, true, _, _, false, true, false, true, ICMP6_ECHO, 0 ) : ping6_reply;
-			( 0, false, false, true, _, _, false, true, false, true, ICMP6_ECHO, _ ) : drop_bad_ping;
-			( 0, false, false, _, USER_SPACE_SERVICE_PORT, true, _, _, _, _, _, _ ) : forward_from_userspace;
-			( 0, false, true, _, USER_SPACE_SERVICE_PORT, true, _, _, _, _, _, _ ) : forward_from_userspace;
-			( 0, false, false, _, _, false, true, _, _, _, _, _ ) : forward_to_userspace;
-			( 0, false, false, true, _, _, _, _, _, _, _, _ ) : forward_to_userspace;
-			( 0, false, true, _, _, _, _, _, _, _, _, _ ) : multicast_inbound;
+			( 0, false, false, _, true, _, _, false, true, true, false, ICMP_ECHOREPLY, 0 ) : forward_to_userspace;
+			( 0, false, false, _, true, _, _, false, true, true, false, ICMP_ECHOREPLY, _ ) : drop_bad_ping;
+			( 0, false, false, _, true, _, _, false, true, true, false, ICMP_ECHO, 0 ) : ping4_reply;
+			( 0, false, false, _, true, _, _, false, true, true, false, ICMP_ECHO, _ ) : drop_bad_ping;
+			( 0, false, false, _, true, _, _, false, true, false, true, ICMP6_ECHOREPLY, 0 ) : forward_to_userspace;
+			( 0, false, false, _, true, _, _, false, true, false, true, ICMP6_ECHOREPLY, _ ) : drop_bad_ping;
+			( 0, false, false, _, true, _, _, false, true, false, true, ICMP6_ECHO, 0 ) : ping6_reply;
+			( 0, false, false, _, true, _, _, false, true, false, true, ICMP6_ECHO, _ ) : drop_bad_ping;
+			( 0, false, false, _, _, USER_SPACE_SERVICE_PORT, true, _, _, _, _, _, _ ) : forward_from_userspace;
+			( 0, false, true, true, _, USER_SPACE_SERVICE_PORT, true, _, _, _, _, _, _ ) : forward_from_userspace;
+			( 0, false, false, _, _, _, false, true, _, _, _, _, _ ) : forward_to_userspace;
+			( 0, false, false, _, true, _, _, _, _, _, _, _, _ ) : forward_to_userspace;
+			// Link-local multicast
```

Comment:
Since we're changing quite a bit around link-local multicast, we need to test this e2e with ddm.